### ISSUE TITLE
[Bug]: Cyborg instillation of an active Positronic brain disappears

### ISSUE THREAD

**Issue by ARandoDude1337** [Aurorastation/Aurora.3#16273]
### Checks

- [X] I have searched the bug with a few keywords, and I confirm this bug was not yet reported.
- [X] The round I am reporting the bug from, or I am going to talk about, has already ended.

### Location

Horizon

### Describe the issue

When creating a cyborg with a positronic brain, after the set up is done and i install the brain. The borg disappears from vision, On inspection even a right click doesn't show it poping up. It can interact with other things and creatures but is entirely invisible and unable to be given any modules or assistance. it cannot be targeted harmed or moved in any way from my experimentation as it simply doesn't exist but yet occupies the space it is in. 

I have heard that this only happens later in the rounds but I haven't had any confirmation of that from personal experience.

TLDR. Making a Borg with a player Brain doesn't work and sends them to the blue space equivalent of Detroit. Please help.

### How to reproduce

1. Create a complete borg shell
2. get a player to enter a postrionic brain (Not an MMI i've seen those work)
3. slot brain into the borg shell
4. Poof. It no longer exsists
5. Profit (If it works the first time with no issues i recommend waiting for a bit incase my ("Later game" broken theory I heard is true and borgs only stop working after the "Start" of the shift whenever that ends i couldn't tell you)

### Round ID

cnA-bEr8

**Comment by Melariara** [comment]
To add to this from the view variables perspective, it changes their set_invisibility to something like 121 or something crazy.
